docs/decisions/backend/0010-advanced-installer-suggestions.md

```diff
@@ -0,0 +1,81 @@
+# Advanced Installer: Suggestions
+
+This document describes the design of the 'Suggestions' system within Advanced Installer, pictured
+in the following mockup below:
+
+![](./images/0009-advanced-installer-location.png)
+
+## Context Statement
+
+When using the Advanced Installer, the user should be provided with 'hints' dictating where a file might
+require to be placed based on a number of heuristics.
+
+This is part of the UX effort to 'Make Modding Easy' and a general requirement of our [Advanced Installer Design](./0009-advanced-installer-design.md).
```

Comment:
The purpose of the Suggested locations area is not to suggest possible locations but rather list *All * the available locations.
"Suggested locations" is just a user facing name for it.

So the objective here isn't to help the user choose, but rather to offer all the locations they might want/need to install the files to. 

Intelligent sorting of the options based on potential knowledge is outside the scope for now.

Reply:
> Game folder should always be suggested, if Appdata and Mygames exist for that game install, they should be suggested as well, even if the game does not declare any InstallFolderTarget for them.

I agree, however the designs don't currently accomodate this, so feedback from Design ( @captainsandypants ) is requested.

The current designs assume the only output folder is game directory. 

e.g. From a style perspective, are we supposed to make separate bordered regions for other types of output paths?

Reply:
You just add the entry here:
![image](https://github.com/Nexus-Mods/NexusMods.App/assets/26797547/59226c5f-a3c5-413b-b3f7-cd9524248fa4)

For example for starfield you would add MyGames/Starfield as another target location.
For Baldur's Gate 3 you would have Appdata/Larian and the Game folder.

So you might need some short name or description of the Target paths from the Games.

You would add another expandable section here for MyGames, under the SkyrimSpecialEdition section.
![image](https://github.com/Nexus-Mods/NexusMods.App/assets/26797547/2e43cb48-b330-4c1d-9147-5fa08f99f7f1)

We discussed this during some of the design meetings we had for this.

Reply:
Alright, that works.